app/src/main/java/com/android/solvit/shared/model/provider/Provider.kt

```diff
@@ -7,15 +7,15 @@ import com.google.firebase.Timestamp
 data class Provider(
     val uid: String = "",
     val name: String = "",
-    val service: Services = Services.TUTOR,
+    var service: Services = Services.TUTOR,
     val imageUrl: String = "",
-    val companyName: String = "",
-    val phone: String = "",
-    val location: Location = Location(0.0, 0.0, ""),
+    var companyName: String = "",
+    var phone: String = "",
+    var location: Location = Location(0.0, 0.0, ""),
     val description: String = "",
     val popular: Boolean = false,
     val rating: Double = 0.0,
     val price: Double = 0.0,
     val deliveryTime: Timestamp = Timestamp.now(),
-    val languages: List<Language> = emptyList()
```

Comment:
You shouldn't use a var here, as this class is intended to mirror firebase data we should not be able to modify it from the class

Reply:
No, because we can modify after